### ISSUE TITLE
Back references to component

### ISSUE THREAD

**Issue by vanrez-nez** [madebyraygun/component-library#27]
Fractal features a small list of other components referencing current component. @daltonrooney I'm thinking this could be useful to have but I'm not entirely sure if you have use this before?

![image](https://github.com/user-attachments/assets/e6f90bd0-8407-446b-b3b1-720bd87a8efb)

I've used this occasionally and I remember having dependencies traced was useful to review what could be broken.

An idea for the backburner: we could also have a full dependency graph page for all components in the library.

**Comment by daltonrooney** [comment]
Yes! I have definitely used this feature before, it’s extremely useful. 👍🏻
